Source/Adobe.Target.Client/TargetClient.cs

```diff
@@ -45,6 +45,7 @@ public static TargetClient Create(TargetClientConfig clientConfig)
         public void Initialize(TargetClientConfig clientConfig)
         {
             this.targetService = new TargetService(clientConfig);
+            this.localService = new OnDeviceDecisioningService(clientConfig);
             this.defaultDecisioningMethod = clientConfig.DecisioningMethod;
             this.defaultPropertyToken = clientConfig.DefaultPropertyToken;
```

Comment:
Please remove `Console.WriteLine`, use logging for debugging.

Reply:
Fixed